javascript/processingFunctions/replaceTagWithSymbol.js

```diff
@@ -34,8 +34,23 @@ const tagsToReplace = {
   FIXED_SPACE: "{\\tt~}"
 };
 
-export const replaceTagWithSymbol = (node, writeTo) => {
+const tagsToReplacePdf = {
+  SHORT_SPACE: "@xxx", // will be replaced in processSnippet
+  SHORT_SPACE_AND_ALLOW_BREAK: "@yyy" // will be replaced in processSnippet
+};
+
+export const replaceTagWithSymbol = (node, writeTo, type = "default") => {
```

Comment:
Add parameter type. `SHORT_SPACE` and `SHORT_SPACE_AND_ALLOW_BREAK` are replaced if `type === "pdf"`